### DISCUSSION TITLE
Welcome to remnote-plugins Discussions!

### DISCUSSION THREAD

**Discussion by browneyedsoul** [browneyedsoul/remnote-plugins#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
How to create a table after adding the CSS block?

I've added the Modern Table Column to the Custom CSS block. 
So now how do I use (call) it in the Document?
